libnavigation-ui/src/main/assets/sdk_versions/com.mapbox.navigation.ui

```diff
@@ -0,0 +1,2 @@
+libnavigation-ui/1.5.0-SNAPSHOT
```

Comment:
This file needs to be deleted as it's adding a new module `libnavigation-ui`

Reply:
removed